wasm/__init__.py

```diff
@@ -4472,14 +4478,7 @@ def init_store():
 
 
 def decode_module(bytestar):
-    try:
-        mod = spec_binary_module(bytestar)
-    except AssertionError:
-        raise
-    except EXCEPTIONS_TO_RERAISE:
-        raise
-    except Exception as err:
-        return "malformed"
+    mod = spec_binary_module(bytestar)
```

Comment:
Sweet, maybe return in the same line.

Looks like `decode_module` is an alias for `spec_binary_module` now, which might have other knock-on effects (not for this PR).